src/main/java/karen/tasks/TaskList.java

```diff
@@ -0,0 +1,88 @@
+package karen.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    public Task getTask(int i) {
+        return this.tasks.get(i);
+    }
+
+    public int getSize() {
+        return this.tasks.size();
+    }
+
+    public void addTask(Task t) {
+        tasks.add(t);
+    }
+
+    public void deleteTask(int i) {
+        tasks.remove(i);
+    }
+
+    public void markTask(int i) {
+        tasks.get(i).mark();
+    }
+
+    public void unmarkTask(int i) {
+        tasks.get(i).unmark();
+    }
+
+    /**
+     * Returns an array of all the taskStrings for each <code>Task</code> in the <code>TaskList</code>
+     * @return String[]
+     */
+    public String[] toTaskStrings() {
+        if (this.tasks.isEmpty()) {
+            return new String[] {};
+        } else {
+            String[] taskStrings = new String[this.tasks.size()];
+            for (int i = 0; i < this.tasks.size(); i++) {
+                taskStrings[i] = this.tasks.get(i).toString();
+            }
+            return taskStrings;
+        }
+    }
+
+    /**
+     * Returns a {@code List<String>} containing each Task's fileString
+     * @return {@code List<String>}
+     */
+    public List<String> toFileStrings() {
+        if (this.tasks.isEmpty()) {
+            return new ArrayList<>();
+        } else {
+            List<String> fileStrings = new ArrayList<>();
+            for (Task t : tasks) {
+                fileStrings.add(t.toFileString());
+            }
+            return fileStrings;
+        }
+    }
```

Comment:
Great use of method names! Very intuitive since it follows the `toString` style. However, why is there an inconsistency of return type between the two functions?